components/sysutils/nut/Makefile

```diff
@@ -104,9 +104,9 @@ CONFIGURE_OPTIONS +=	--with-macosx_ups=no
 
 # Note: Link with the correct ISA of net-snmp (for drivers) and gdlib (for cgi)
 CONFIGURE_OPTIONS +=	--with-snmp
-CONFIGURE_OPTIONS +=	--with-net-snmp-config=$(USRBINDIR)/net-snmp-config-$(BITS)
 
-CONFIGURE_OPTIONS.32 +=	--with-gdlib-config=$(USRBINDIR)/gdlib-config
+CONFIGURE_OPTIONS.32 +=	--with-net-snmp-config=$(USRBINDIR)/net-snmp-config-32
```

Comment:
Why should it be changed?

Reply:
I don't like it too but with this change it is the same as in asterisk.
I don't mind to use the net-snmp-config in /usr/bin/ or /usr/bin/amd64 respective.
But we should do it the same way when net-snmp-config is used.